.github/workflows/ci_cd_updated_master.yml

```diff
@@ -5,57 +5,57 @@ on:
     branches: [master]
 
 env:
-  # DEPENDABOT_BRANCH: ci/dependabot-updates
+  DEPENDABOT_BRANCH: ci/dependabot-updates
   GIT_USER_NAME: "TEAM 4.0[bot]"
   GIT_USER_EMAIL: "Team4.0@SINTEF.onmicrosoft.com"
```

Comment:
```suggestion
  GIT_USER_EMAIL: "Team4.0@SINTEF.no"
```

Reply:
Ah - that works now? 😮

Reply:
Yes, the test I sent yesterday was that. Sorry for not notifying you this morning.

Reply:
Excellent! Let's update it everywhere then where the "old" email has been used. I'll update here as well of course.